code/modules/cargo/packs.dm

```diff
@@ -1980,6 +1980,7 @@
 					/obj/item/toy/plush/lizardplushie,
 					/obj/item/toy/plush/snakeplushie,
 					/obj/item/toy/plush/nukeplushie,
+					/obj/item/toy/plush/goatplushie/angry,
```

Comment:
make the plushiecrate have the non-angry version and i'll merge it, not gonna merge it now since it makes the plushiecrate attack people when opened.